### PR TITLE
Slightly adjusted the readme.md

### PR DESCRIPTION
**Pull request by moritz-scandit**
(cherry picked from commit a22b53f2585fda4918f53d9e4bfbcce75af6e4f1)

### CODE REVIEW

File: samples/extended/readme.md [hunk 1]
```diff
@@ -13,6 +13,22 @@ Run from a different folder
...
+```
+> phonegap platform add ios
+> phonegap run ios --device
+```
+
+For Windows:
+```
+> phonegap platform add windows
+> phonegap run windows --device
+```
```
Comment (lucatorella) [inline samples/extended/readme.md]: I think we should switch from `phonegap` to `cordova`. If I do `phonegap ...` it says `phonegap: command not found` :)

Comment (moritz-scandit) [reply samples/extended/readme.md]: do you not have phonegap installed then? :P

Comment (lucatorella) [reply samples/extended/readme.md]: can you still install it? they renamed the cli to cordova quite some time ago :)

Comment (moritz-scandit) [reply samples/extended/readme.md]: yea you can, i think there is a naming question outstanding between cordova and phonegap, but let's not do it right now :)

Comment (lucatorella) [reply samples/extended/readme.md]: 🤦🏻‍♂️

**Pull request merged**